analytics_data_api/v0/views/courses.py

```diff
@@ -653,7 +653,11 @@ def get_queryset(self):
                 # http://code.openark.org/blog/mysql/those-oversized-undersized-variables-defaults.
                 cursor.execute("SET @@group_concat_max_len = @@max_allowed_packet;")
 
-            cursor.execute(sql, [self.course_id])
+            try:
+                cursor.execute(sql, [self.course_id])
+            except OperationalError:
```

Comment:
I'm curious about catching this exception.  Is it possible to determine which column should be specified before we hit this?